### PR TITLE
Add github action to run tests.

### LINKED ISSUE #136: Migrate from Travis to GitHub Actions

### PR DESCRIPTION
**Pull request by valberg**
Closes #136

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -13,4 +19,4 @@ deps =
     django220: Django>=2.2.0,<2.3.0
     django300: Django>=3.0.0,<3.1.0
     django310: Django>=3.1.0,<3.2.0
-    djangolatest: https://github.com/django/django/archive/master.tar.gz
```
Comment (fjsj) [inline tox.ini:16]: Can we keep the djangolatest? It's useful for us to know beforehand if something will break in the next Django release.

Comment (valberg) [reply tox.ini:16]: Sure - it just didn't work when I ran the action locally with act. I'll give it another try 😉

Comment (valberg) [reply tox.ini:16]: @fjsj it's back in - although it does fail ;)

**Comment by fjsj** [comment]
Thank you very much, merging.

**Pull request merged**